spec/javascripts/NewService/NewServiceForm.spec.jsx

```diff
@@ -1,8 +1,14 @@
+// @flow
+
 import React from 'react'
 import Enzyme, {mount} from 'enzyme'
 import Adapter from 'enzyme-adapter-react-16'
 
-import {NewServiceForm} from 'NewService'
+import {NewServiceForm, ServiceManualForm, ServiceDiscoveryForm} from 'NewService'
+
+import * as utils from 'utilities/utils'
+jest.spyOn(utils, 'CSRFToken')
+  .mockImplementation(() => '')
```

Comment:
Instead of mocking the implementation in every file, maybe we could mock the module

Reply:
Yes 100% agree. But in order not to get too much out of scope with this PR I created this [other ticket](https://issues.jboss.org/browse/THREESCALE-3104) for that and other repetitive setups.